src/providers/profile/profile.ts

```diff
@@ -1438,60 +1428,74 @@ export class ProfileProvider {
     };
   }
 
-  public createDefaultWallet(coins): Promise<any> {
+  public createTokenWallet(ethWallet, token) {
+    if (_.isString(token)) {
+      let tokens = this.currencyProvider.getAvailableTokens();
+      token = tokens.find(x => x.symbol == token);
+    }
+    const tokenCredentials = ethWallet.credentials.getTokenCredentials(token);
+    const walletClient = this.bwcProvider.getClient(null, {
+      baseUrl: ethWallet.baseUrl,
+      bp_partner: ethWallet.bp_partner,
+      bp_partner_version: ethWallet.bp_partner_version
+    });
+    walletClient.fromObj(tokenCredentials);
+    return walletClient;
+  }
+
+  public createMultipleWallets(coins: string[], tokens = []): Promise<any> {
     return new Promise((resolve, reject) => {
+      if (tokens && tokens.length && coins.indexOf('eth') <= 0) {
```

Comment:
if only eth is selected then:
```
coins = ['eth']` then index will be `0`. 
```

<img width="307" alt="Screen Shot 2019-11-15 at 1 26 54 PM" src="https://user-images.githubusercontent.com/23103037/68966407-ff2a8480-07ab-11ea-9a57-b61285d5acfb.png">

Reply:
should be `tokens.length = 0 && && coins.indexOf('eth') < 0`

Reply:
I also got that popup too, but the wallets (ETH and USDC selected) were created anyway

Reply:
thanks!

      if (tokens && tokens.length && coins.indexOf('eth') < 0) {

if should fail if no `eth` is given a parameter and one or more token is given.